### ISSUE TITLE
Implement all react-native components

### ISSUE THREAD

**Issue by flexsurfer** [status-im/status-fiddle#7]

**Comment by status-open-bounty** [comment]
![Contract deploying](https://openbounty.status.im/img/deploying_contract.png)
Transaction: [0x68514d54b14af58f9aac4057260eb27fc3140d66336afceac5ba6e16a5a94b06](https://etherscan.io/tx/0x68514d54b14af58f9aac4057260eb27fc3140d66336afceac5ba6e16a5a94b06)

**Comment by Monte9** [comment]
Hey @flexsurfer! I came across this issue/repo when I am digging into https://status.im/ and found it really interesting. 

I have plenty of React Native experience and I would like to help out with this issue is possible. Can you please let me know how I can help?

Also, I have very little context on how this repo connects to the main status ios/android app and the related tech. Could you kindly give me a tldr; on the status ecosystem? 

Let me know. Thanks.

**Comment by audriu** [comment]
@flexsurfer If I understand right - here is needed to port all components from `status-react` project like ` (def activity-indicator (get-class "ActivityIndicator"))' [documented](https://facebook.github.io/react-native/docs/activityindicator.html) ? and make them work.

**Comment by flexsurfer** [comment]
yes @audriu you're right, all components used in status-react and supported here https://github.com/necolas/react-native-web/blob/master/packages/react-native-web/src/index.js#L31, and `status-im.ui.components.list.views`  and `status-im.ui.components.status-bar.view`

**Comment by flexsurfer** [comment]
let's say these components , functions and ns should be implemented

```
(def activity-indicator (get-class "ActivityIndicator"))

(defn keyboard-avoiding-view 
(def keyboard-avoiding-view-class (get-class "KeyboardAvoidingView"))

(def status-bar (get-class "StatusBar"))
status-im.ui.components.status-bar.view

(def flat-list-class (react/get-class "FlatList"))
(def section-list-class (react/get-class "SectionList"))
status-im.ui.components.list.views
```

**Comment by audriu** [comment]
I am working on!

**Comment by audriu** [comment]
What if to symlink whole `/status-fiddle/src/cljs/status_im` instead of copying peace by peace?

**Comment by audriu** [comment]
@flexsurfer  What if to symlink whole `/status-fiddle/src/cljs/status_im` instead of copying peace by peace?

**Comment by flexsurfer** [comment]
@audriu, in this case, we need to compile all status_im, but we don't have all dependencies and don't want to have

**Comment by flexsurfer** [comment]
i'm working on this

**Comment by pablanopete** [comment]
To be revoked cc @andytudhope

**Comment by StatusSceptre** [comment]
@flexsurfer is this still  relevant to bounty?

**Comment by flexsurfer** [comment]
no, we can close it
